### PR TITLE
Added Scripts to Join Lines by different Delimiters

### PR DESCRIPTION
**Pull request by riesentoaster**
Added 3 scripts to join lines without a delimiter, by a space or by a comma.

### CODE REVIEW

File: Boop/Boop/scripts/JoinLinesWithSpace.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+		"description":"Joins All Lines With a Space",
+		"author":"riesentoaster",
+		"icon":"table",
+		"tags":"join, space",
+        "bias": -0.1
+	}
+**/
+
+function main(input) {
+	input.text = input.text.replace('\n', ',');
```
Comment (spoike) [inline Boop/Boop/scripts/JoinLinesWithSpace.js:14]: Might be a good idea to actually replace with a space as specified. :-)
```suggestion
	input.text = input.text.replace('\n', ' ');
```

File: Boop/Boop/scripts/JoinLines.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/**
+	{
+		"api":1,
+		"name":"Join Lines",
+		"description":"Joins All Lines Without a Delimiter",
```
Comment (spoike) [inline Boop/Boop/scripts/JoinLines.js:5]:
```suggestion
		"description":"Joins All Lines Without any Delimiter",
```

File: Boop/Boop/scripts/JoinLinesWithComma.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/**
+	{
+		"api":1,
+		"name":"Join Lines With Comma",
+		"description":"Joins All Lines With a Comma",
+		"author":"riesentoaster",
+		"icon":"table",
+		"tags":"join, comma",
+        "bias": -0.1
```
Comment (spoike) [inline Boop/Boop/scripts/JoinLinesWithComma.js:9]: whitespace for bias and the rest of the data is inconsistent

File: Boop/Boop/scripts/JoinLinesWithSpace.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+/**
+	{
+		"api":1,
+		"name":"Join Lines With Space",
+		"description":"Joins All Lines With a Space",
+		"author":"riesentoaster",
+		"icon":"table",
+		"tags":"join, space",
+        "bias": -0.1
```
Comment (spoike) [inline Boop/Boop/scripts/JoinLinesWithSpace.js:9]: whitespace for bias and the rest of the data is inconsistent (part 2)

Comment (riesentoaster) [reply Boop/Boop/scripts/JoinLinesWithSpace.js]: Fixed the whitespace/tab issue.

**Comment by riesentoaster** [comment]
I'm not really sure why the second test didn't pass. To my knowledge, I fixed the issues with the code, sorry about them.

**Comment by IvanMathy** [comment]
Hey there! PR looks good, I think you'll need to pull the main branch since the dependency system changed...

**Comment by riesentoaster** [comment]
So I pulled your master branch to a local clone of my fork, merged them and pushed it back to my fork on GitHub. Is this correct?

(Sorry about all the questions, this is my first pull request.)

**Comment by IvanMathy** [comment]
Yep, that's perfect! 

Thanks for the contribution, great job and congrats on your first PR 👏 👏 👏

**Pull request merged**